chore(amazon): add zen 3 to us, es, and nl

picchioni opened the pull request:
### Description

Fixes #740 in addition to adding missing Ryzen CPUs to other regional Amazon sites

### Testing

1. Navigate to various Amazon product pages to confirm product exists and add to card functionality works

### New dependencies

None

jef commented on src/store/model/amazon-es.ts:
```diff
@@ -99,6 +99,34 @@ export const AmazonEs: Store = {
+			cartUrl: 'https://www.amazon.es/gp/aws/cart/add.html?ASIN.1=B08166SLDF&Quantity.1=1',
+			model: '5600x',
+			series: 'ryzen5600',
+			url: 'https://www.amazon.es/dp/product/B08166SLDF'
```
```suggestion
			url: 'https://www.amazon.es/dp/B08166SLDF'
```

Do you mind cleaning this up for the rest of the URLs?

picchioni replied:
@jef Sure thing, I'll get that cleaned up later this evening.

jef replied:
Thanks!

jef reviewed (changes requested):
Just a small change, otherwise looks great! I appreciate the contribution!

picchioni commented:
Hey @jef I cleaned up the URLs, and added cartURL for outstanding products across all Amazon regions. I noticed after committing, the files use tabs instead of spaces for indentation which threw off the formatting. I was going to convert the tabs to 4 spaces per tab but wanted to check with you first, otherwise I'll convert my code to tabs. Thoughts?

jef commented:
The linter will fix this for you. Just run `npm run lint:fix`
